src/web/app/user-join-page.component.html

```diff
@@ -12,7 +12,8 @@ <h3>Not logged in</h3>
       <h3>Invalid course join link</h3>
     </div>
     <div class="card-body" *ngIf="!validUrl">
-      You are using an invalid join link. Please <a [routerLink]="link">request for a new join link</a> or check that you copied the URL entirely.
+      You are using an invalid join link. Please <a routerLink="/web/front/request">request for a new join link</a>
+      or check that you copied the URL entirely.
```

Comment:
Requesting a new join link is not the same as requesting a new account; the latter creates a duplicate sample course.

Reply:
So should I remove the redirection to the request page then? I'll create an issue separately for regenerating join links